src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {

```

Comment:
Don't begin class implementation with an empty line. Please remove the empty line before the class declaration.